README.md

```diff
@@ -1,2 +1,5 @@
-# Data-Structures-and-Algorithms \
-So this repo is a collection of great mini projects on data structures and algorithms.
\ No newline at end of file
+# Data-Structures-and-Algorithms 
+So this repository is a collection of great mini projects on data structures and algorithms. The aim of the repository is to demonstrate and apply the algorithms in simple and efficient way.
```

Comment:
Remove So and Capitalize 'T' in this

Reply:
Sure!

Reply:
Also, I am new to this can you please tell that how will I update my pull request since I updated what you told.